sdk/quantum/Azure.Quantum.Jobs/README.md

```diff
@@ -74,12 +77,22 @@ var quantumJobClient =
         credential);
 ```
 
+### Get Container SAS URI
+
+Create a storage container where to put your data.
+
 ```C# Snippet:Azure_Quantum_Jobs_GetContainerSasUri
 // Get container Uri with SAS key
 var containerUri = (quantumJobClient.GetStorageSasUri(
     new BlobDetails(storageContainerName))).Value.SasUri;
 ```
 
+### Upload Input Data
+
+Using the SAS URI, upload the json input data to the blob client.
+This contains the parameters to be used with [Quantum Inspired Optimizations](https://github.com/MicrosoftDocs/quantum-docs-private/wiki/Optimization-quickstart-for-Azure-Quantum)
```

Comment:
quantum-docs-private in the URI makes me think people won't have access to these?